includes/discovery/sensors/current/mgeups.inc.php

```diff
@@ -3,7 +3,7 @@
 // MGE UPS
 if ($device['os'] == 'mgeups') {
     echo 'MGE ';
-    $oids = trim(snmp_walk($device, '1.3.6.1.4.1.705.1.7.1', '-OsqnU'));
+    $oids = trim(snmp_walk($device, '1.3.6.1.4.1.705.1.7.2.1.5', '-OsqnU'));
```

Comment:
Why did you change the OIDs?  Do the new OIDs support other devices such as Galaxy and Evolution?